internal/pkg/externalplugins/config.go

```diff
@@ -422,3 +473,41 @@ func validateBlunderbuss(blunderbusses []TiCommunityBlunderbuss) error {
 
 	return nil
 }
+
+// validateLabelBlocker will return an error if the regex cannot compile or actions is illegal.
+func validateLabelBlocker(labelBlockers []TiCommunityLabelBlocker) error {
```

Comment:
Should we check if teams and users are empty?

Reply:
I think it doesn't feel necessary.

Reply:
So how should it work if I don't configure it?

Reply:
If there are no trusted users and trusted teams, the operation of everyone is limited.

In the future, we can expand to realize that a certain label can only be used on PR or issue.

Reply:
But then even robots can't add a label? The label can never be added.

Reply:
What I mean is that if a label is restricted to be used on the issue, everyone including robots cannot be used on pr, but it will not be restricted on the issue.

Reply:
Got it.